### PR TITLE
Exercise 14

### PR DESCRIPTION
**Pull request by cristianodeoliveira**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -118,3 +118,24 @@ env/
...
+.terraform
+
+# Ignore minikube
+minikube-darwin-amd64*
+
+#Ignore my local deployment .yaml
+#deployment.yaml
+
+#ignore secret.yaml
+secrect.yaml
```
Comment (RayCoulstock) [inline .gitignore]: There is a typo here Cristiano.  You've spelt Secret wrong and so it isn't ignoring the secret.yaml.  In fact it is storing your secrets file in GIT. It's a simple mistake but it now means there is a potential security breach as I can see your secret API keys.

**Review by RayCoulstock — CHANGES_REQUESTED** [review]
Once you have fixed your gitignore file and spelt secret.yaml correctly it all looks good to me.

**Comment by cristianodeoliveira** [comment]
hi Ray, I don't know how to resolve the conflicts I am afraid and I wonder if we could have a call to talk about it so that I can complete this part of the training?

**Comment by RayCoulstock** [comment]
Hi Cristiano,

I’m always happy to have chat.  This might help [Deleting files in a repository - GitHub Docs](https://docs.github.com/en/repositories/working-with-files/managing-files/deleting-files-in-a-repository).  It can be a pain to remove files from a GitHub repo history.  Don’t worry too much about it this time as it is only a learning exercise and no one else is looking at it beyond me.

If you still want to book a slot for a chat, feel free or wait until we have our 1-2-1 this month.

Cheers,
Ray.

**Comment by cristianodeoliveira** [comment]
Hi Ray,All resolved now and code merged.CrisSent from my iPhoneOn 5 Nov 2024, at 13:25, Ray Coulstock ***@***.***> wrote:﻿
Hi Cristiano,
I’m always happy to have chat.  This might help Deleting files in a repository - GitHub Docs.  It can be a pain to remove files from a GitHub repo history.  Don’t worry too much about it this time as it is only a learning exercise and no one else is looking at it beyond me.
If you still want to book a slot for a chat, feel free or wait until we have our 1-2-1 this month.
Cheers,
Ray.

**Pull request closed without merging**